File: running-sum-of-1d-array/running-sum-of-1d-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    var sum=0;
    var sumArray=[];
    for (let i=0;i<nums.length;i++){
        sum=sum+nums[i]
        sumArray.push(sum)
    }
    return sumArray;
};

File: running-sum-of-1d-array/running-sum-of-1d-array.java
import java.util.*;
class Solution {
    public int[] runningSum(int[] nums) {
        ArrayList<Integer> list=new ArrayList<Integer>();
    int sum=0;
    for(int i=0;i<nums.length;i++){
        sum=sum+nums[i];
        list.add(sum);
    }
       
        Object[] array = list.toArray();
         int[] intArray = new int[array.length];
        for(int i=0;i<array.length;i++){
            intArray[i]=(int)array[i];
        }
    return intArray;
        }
    }
​
